docs/generators/csv.rst

```diff
@@ -1,6 +1,21 @@
 CSV
 ========
+This generator outputs the classes of the LinkML schema in CSV format. Each row contains the identifier, mappings and description of each class in the schema.
 
+This generator does not output a CSV that is configured based on the template. If fillable template spreadsheets are required, consider using [gen-excel](https://linkml.io/linkml/generators/excel.html) or [DataHarmonizer](https://github.com/cidgoh/DataHarmonizer). 
+
+Auto-generated docs
+-------------------
+
+Code
+^^^^
 .. automodule:: linkml.generators.csvgen
     :members:
     :undoc-members:
+
+Command Line
+^^^^^^^^^^^^
```

Comment:
This section looks odd to me. Are those ^^^^^ supposed to be there?

Reply:
@timalamenciak can you take a look and tell me what is going on here? I'm not all that familiar with .rst either.

Reply:
Yes -- that is valid rst header syntax. See: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections